Tests/FunctionalTests/AEPEdgeFunctionalTests.swift

```diff
@@ -34,8 +34,8 @@ class AEPEdgeFunctionalTests: FunctionalTestBase {
         continueAfterFailure = false
         FileManager.default.clearCache()
 
-        // hub shared state update for 2 extension versions (InstrumentedExtension (registered in FunctionalTestBase), IdentityEdge, Edge) IdentityEdge XDM and Config shared state updates
-        setExpectationEvent(type: FunctionalTestConst.EventType.HUB, source: FunctionalTestConst.EventSource.SHARED_STATE, expectedCount: 4)
+        // hub shared state update for 1 extension versions (InstrumentedExtension (registered in FunctionalTestBase), IdentityEdge, Edge) IdentityEdge XDM and Config shared state updates
```

Comment:
Isn't this expected to have two shared state updates when calling:
MobileCore.registerExtension(ExtensionExample.self)
MobileCore.registerExtensions([ExtensionExample2.self, ExtensionExample3.self])

Reply:
Corrected: two shared state updates are expected for:
```
MobileCore.registerExtensions([ExtensionExample.self])
MobileCore.registerExtensions([ExtensionExample2.self, ExtensionExample3.self])
````
but `registerExtension` does not start the core yet.